### ISSUE TITLE
Custom date modes?

### ISSUE THREAD

**Issue by Jonatthu** [farreldarian/prisma-generator-drizzle#45]
Default output is:
```
  latestMessageCreatedAt: timestamp('latestMessageCreatedAt', { mode: 'date', precision: 3 })
    .defaultNow()
    .notNull(),
```
How do I change it to:

```
  latestMessageCreatedAt: timestamp('latestMessageCreatedAt', { mode: 'string', precision: 3 })
    .defaultNow()
    .notNull(),
```

or modify the options?

**Comment by farreldarian** [comment]
This can be done

**Comment by farreldarian** [comment]
@Jonatthu, apologize for the delayed response. I have just released version [0.7.0](https://github.com/farreldarian/prisma-generator-drizzle/releases/tag/v0.7.0). Kindly take a look at it.

**Comment by Jonatthu** [comment]
No apologies needed, indeed thanks a lot for the changes and the feature!

**Comment by Jonatthu** [comment]
Anyway just to change it at the field level instead of global level?

**Comment by farreldarian** [comment]
> Anyway just to change it at the field level instead of global level?

Released [0.7.1](https://www.npmjs.com/package/prisma-generator-drizzle/v/0.7.1) with field-level support, docs coming soon

```
model User {
  ...
  /// drizzle.dateMode string
  stringDate DateTime
  ...
}
```

**Comment by Jonatthu** [comment]
Just tested it and it's working fine!
